### DISCUSSION TITLE
Doubts how to type biblical references

### DISCUSSION THREAD

**Discussion by MarlusMM** [tim-hub/obsidian-bible-reference#62] [Q&A] [answered]
How to type in Obsidian, using the Bible Reference Plugin, duplicate books such as 1 and 2 Corinthians, and books that contains only one chapter, such as Jude? 
I Use Portuguese version - Almeida: João Ferreira de Almeida
@tim-hub 
#66

**Comment by JPT62089** [comment, marked as answer]
Hey @MarlusMM! Sorry for the delayed response on this. Unfortunately at this time I believe all of the commands need to be in English or abbreviations of English books. So using the books you mentioned:

--2Cor3:16
--2Corinthians3:16
--jude1:2

The above should work :)
